src/checkly/models.ts

```diff
@@ -279,3 +279,11 @@ export class CheckStatus {
   sslDaysRemaining: number | null;
   isDegraded: boolean;
 }
+
+export class CheckSyncStatus {
```

Comment:
General question: do we need to persist this as all? Isn't querying the last check result from db enough to get the timeframe?

Reply:
Additional point: can't we infer the type from knex?